### PR TITLE
Added currency converter

### LINKED ISSUE #183: Add currency converter
Currency converter with html ,css and javascript

currency of one country must be converted to other

image

### PR DESCRIPTION
**Pull request by pranav514**
# Title and Issue number 

Title :currency-converter

Issue No. :#183

Code Stack : html,css ,javascript

Close #183 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

File: native/machine-coding/currency-converter/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="../../helpers/navbar.js" type="module"></script>
+    <title>Currency Converter</title>
```
Comment (sadanandpai) [inline native/machine-coding/currency-converter/index.html:8]: add header.js only to the head (as per the new standards)
refer [counter](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/native/machine-coding/counter/index.html)

File: native/machine-coding/currency-converter/index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+<body>
+    <div class="converter">
+        <h1>Currency Converter</h1>
+        <div class="input-container">
+            <input type="number" id="amount" placeholder="Enter amount">
+            <select id="fromCurrency">
+                <option value="USD">USD</option>
+                <option value="EUR">EUR</option>
+                <option value="GBP">GBP</option>
+                <option value="INR">INR</option>
```
Comment (sadanandpai) [inline native/machine-coding/currency-converter/index.html:19]: can we add these currency options dynamically from javascript

File: native/machine-coding/currency-converter/index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            <input type="number" id="result" placeholder="Result" disabled>
+            <select id="toCurrency">
+                <option value="USD">USD</option>
+                <option value="EUR">EUR</option>
+                <option value="GBP">GBP</option>
+                <option value="INR">INR</option>
+                <!-- Add more currencies here -->
+            </select>
+        </div>
+        <button onclick="convertCurrency()">Convert</button>
```
Comment (sadanandpai) [inline native/machine-coding/currency-converter/index.html:33]: move the click handler to js

File: native/machine-coding/currency-converter/style.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
+@import url(../../helpers/core.css);
```
Comment (sadanandpai) [inline native/machine-coding/currency-converter/style.css:1]: can remove this now

File: native/machine-coding/currency-converter/style.css [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+body {
+    font-family: Arial, sans-serif;
+    background-color: #f7f7f7;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
```
Comment (sadanandpai) [inline native/machine-coding/currency-converter/style.css:11]: do not modify body

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by pranav514** [comment]
@sadanandpai  issue resolved pls review it :)

**Pull request merged**